<launch>
  <param name="map_width" type="int" value="15" />
  <param name="map_height" type="int" value="10" />

  <param name="user_pos_x" type="int" value="14" />
  <param name="user_pos_y" type="int" value="9" />
  <param name="house_pos_x" type="int" value="0" />
  <param name="house_pos_y" type="int" value="7" />
  <param name="pet_pos_x" type="int" value="5" />
  <param name="pet_pos_y" type="int" value="5" />
</launch>